cli/command/trust/info.go

```diff
@@ -101,6 +101,38 @@ func lookupTrustInfo(cli command.Cli, remote string) error {
 	}
 	printSignatures(cli, remote, signatureRows)
 
+	roleWithSigs, err := notaryRepo.ListRoles()
+	if err != nil {
+		fmt.Fprintf(cli.Out(), "Error listing signature metadata for roles: %s\n", err)
+		return err
+	}
+
+	signerRoleToKeyIDs := make(map[string][]string)
+	baseRoleToKeyIDs := make(map[string][]string)
+
+	for _, roleWithSig := range roleWithSigs {
+		// TODO(ashfall) Can we get rid of this for loop? Perhaps find
+		// a notary call that filters for us?
+
+		// roleWithSig.Signatures -> [] data.Signature
+		if data.IsDelegation(roleWithSig.Name) {
+			signerRoleToKeyIDs[notaryRoleToSigner(roleWithSig.Name)] = roleWithSig.KeyIDs
+		} else {
+			baseRoleToKeyIDs[roleWithSig.Name.String()] = roleWithSig.KeyIDs
+		}
+	}
+
+	fmt.Fprintf(cli.Out(), "\nList of signers and their KeyIDs:\n\n")
```

Comment:
similarly, can we move this printing into a helper function for the time being?  It'll make the pretty printing diff simpler.